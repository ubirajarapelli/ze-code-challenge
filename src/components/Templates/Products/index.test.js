import React from 'react';
import { render } from '@testing-library/react'
import { ProductsTemplate } from './'

const renderComponent = () => {
  return render(
    <ProductsTemplate>
      <h1>Products Template</h1>
    </ProductsTemplate>
  )
}

describe('Test from ProductsTemplate Component', () => {
  it('should make a snapshot from ProductsTemplate Component', () => {
    const { container } = renderComponent()
    expect(container).toMatchSnapshot()
  })
})
