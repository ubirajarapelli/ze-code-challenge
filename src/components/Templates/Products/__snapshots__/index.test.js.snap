// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test from ProductsTemplate Component should make a snapshot from ProductsTemplate Component 1`] = `
<div>
  <header
    class="sc-dIsUp bvmyLd"
  >
    <nav>
      <img
        alt="Logotipo Brewlivery"
      />
      <span />
      <button
        class="sc-bqGGPW sc-hBMUJo isrxgy mDayR"
      >
        Login
      </button>
      <button
        aria-label="Ir para a sacola de compras"
        class="sc-bqGGPW sc-fnVZcZ isrxgy laRLoJ"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M5 4H19C19.5523 4 20 4.44771 20 5V19C20 19.5523 19.5523 20 19 20H5C4.44772 20 4 19.5523 4 19V5C4 4.44772 4.44771 4 5 4ZM2 5C2 3.34315 3.34315 2 5 2H19C20.6569 2 22 3.34315 22 5V19C22 20.6569 20.6569 22 19 22H5C3.34315 22 2 20.6569 2 19V5ZM12 12C9.23858 12 7 9.31371 7 6H9C9 8.56606 10.6691 10 12 10C13.3309 10 15 8.56606 15 6H17C17 9.31371 14.7614 12 12 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </nav>
  </header>
  <main
    class="sc-lmgQwP bHHAix"
  >
    <div
      class="sc-iqAclL jjNvRh"
    >
      <h1>
        Products Template
      </h1>
    </div>
  </main>
  <footer
    class="sc-crzoAE lfmNBV"
  >
    <div
      class="sc-iqAclL jjNvRh"
    >
      <p>
        Termos de uso e política de privacidade
      </p>
      <p>
        © 2021 Brewlivery.com
      </p>
      <p>
        Beba com moderção
      </p>
    </div>
  </footer>
</div>
`;
